mpse/src/table.py

```diff
@@ -1,6 +1,11 @@
 def is_nan(string):
     return string != string
 
+# Remove os espaços branco
+def conv(name):
```

Comment:
Quando menos manipulação a gente fizer melhor, Marcos. Sei que é chato ver espaços em branco e outros caracteres no início, mas creio que quando mais fiéis a gente se mantiver ao que vem, menos problemas teremos. Poderia retirar esse strip, por favor?